scripts/actions/toggleStats.js

```diff
@@ -0,0 +1,7 @@
+import * as types from '../constants/ActionTypes';
+
+export function toggleStats() {
+  return {
+      type: types.TOGGLE_STATS
```

Comment:
Since I'm not doing any state modifications, I wasn't entirely sure if the action paradigm was the best way to go about this.

Is there an easier way (e.g.: less boilerplate) for UI actions that modify state but don't have state themselves?